smsdk/client.py

```diff
@@ -622,7 +700,7 @@ def get_machines(self, normalize=True, *args, **kwargs):
             "machine_v1", "get_machines", normalize, *args, **kwargs
         )
 
-    def get_machine_names(self, source_type=None, clean_strings_out=True):
+    def get_machine_names(self, *args, **kwargs):
```

Comment:
You could left the args in

Reply:
This is the standard way of defining positional arguments. If there are no positional arguments passed, it will be come None. No need to set it explicitly.